exercises/grains/runtests.jl

```diff
@@ -3,10 +3,14 @@ using Test
 include("grains.jl")
 
 @testset "On squares" begin
-    @testset "On square $s" for s = UInt64(1):64
-        @test on_square(s) == 2^(s-1)
-        @test total_after(s) == 2^s - 1
-    end
+    @test on_square(1) == 1
+    @test on_square(2) == 2
+    @test on_square(3) == 4
+    @test on_square(4) == 8
+    @test on_square(16) == 32768
+    @test on_square(32) == 2147483648
+    @test on_square(64) == 9223372036854775808
```

Comment:
Perhaps for the sake of teaching the cases that require an `Int128` should be in their own testset, so that there's some "progression": first one has to solve the algorithm part, then making sure it works for the entire domain.

 I'll leave this decision up to you, though, I'm fine with it as it is, too.

Reply:
that's actually a good idea.  Do you mind if I change the names of the testsets?  Perhaps calling the first set, "beginning squares" and then "ending squares" for the later tests?

Reply:
also, if I add a test that's not in the canonical-data should I note that?  I was thinking `total_after(1)` as a possible "edge" case and perhaps `total_after(3)` as a non-edge case since currently the only test for that function is an edge case with the Int128.

Reply:
Feel free to rename them.

Additional test cases don't need to be documented, though if they're potentially relevant to other languages  you can open a PR in the problem-spec repo and add them. Both tests sound reasonable to me.